<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="focusScore_graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1031</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="GraphicsLayoutWidget" name="image_graphicsLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>1031</width>
     <height>841</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>50</y>
     <width>221</width>
     <height>379</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="AFCoffset_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>AFC Offset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="z_doubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="section_spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Section</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="frame_spinBox">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Frame</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="review_pushButton">
      <property name="text">
       <string>Review</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="move_pushButton">
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="livereview_pushButton">
      <property name="text">
       <string>Showing Review</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="softwareaf_pushButton">
      <property name="text">
       <string>SoftwareAutoFocus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="snap_pushButton">
      <property name="text">
       <string>Snap</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="retake_pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="text">
       <string>ReTake</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Review</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Load</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Review&amp;Load</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ClickMode</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="channel_groupBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>410</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>channels</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="channel_verticalLayout"/>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
